all-builds.nix

```diff
@@ -41,6 +41,19 @@ let
     else if lib.isList v then lib.concatMap collect v
     else [];
 
+  # A simple derivation that just creates a file with the names of all
+  # of its inputs. If built, it will have a runtime dependency on all
+  # of the given build inputs.
+  pinBuildInputs = name: buildInputs: (local-self.nixpkgs.releaseTools.aggregate {
```

Comment:
This was removed from reflex-platform. Not sure if intentional

Reply:
Yeah, I wasn't needing them while doing the haskell.nix work. I don't think we need them in reflex-platform anymore but pulling it into the obelisk nix is fine by me